les/api_backend.go

```
@@ -342,6 +343,16 @@ func (b *LesApiBackend) CurrentHeader() *types.Header {
 	return b.eth.blockchain.CurrentHeader()
 }
 
+func (b *LesApiBackend) BuildBlockFromTxs(ctx context.Context, buildArgs *types.BuildBlockArgs, txs types.Transactions) (*types.Block, *big.Int, error) {
+	log.Warn("BuildBlockFromTxs is not supported in light client mode")
```

I might be willing to "fail fast" - `log.Panic(...)` WDYT?